### PR TITLE
Added official quest from kRO 'Jellopy Quest'.

### PR DESCRIPTION
**Pull request by Jedzkie**
[//]: # (**********************************)
[//]: # (** Fill in the following fields **)
[//]: # (**********************************)

[//]: # (Note: Lines beginning with syntax such as this one, are comments and will not be visible in your report!)

### Pull Request Prelude

[//]: # (Thank you for working on improving Hercules!)

[//]: # (Please complete these steps and check the following boxes by putting an `x` inside the brackets _before_ filing your Pull Request.)

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR will be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

[//]: # (Describe at length, the changes that this pull request makes.)

**Affected Branches:** master

[//]: # (Master? Slave?)

**Issues addressed:**

[//]: # (Issue Tracker Number if any.)

### Known Issues and TODO List

[//]: # (Insert checklist here)
[//]: # (Syntax: - [ ] Checkbox)

[//]: # (**NOTE** Enable the setting "[√] Allow edits from maintainers." when creating your pull request if you have not already enabled it.)

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

**Comment by HerculesWSAPI** [comment]
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/1696)

File: npc/quests/arlim_quest.txt [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+//================= Description ===========================================
+//= A quest that trades your 100 Jellopies to 5 Beginner Battle Manuals
+//================= Current Version =======================================
+//= 1.0
+//=========================================================================
+prontera,204,53,4	script	Errand Man	4_M_EINMAN,{
+
+	.@pong = questprogress(9024);
+
+	disable_items;
```
Comment (Asheraf) [inline npc/quests/arlim_quest.txt]: `disable_items();`

File: npc/quests/arlim_quest.txt [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+	if (.@pong == 0) {
+		mes("[Errand Man]");
+		mes("aaah, why I..??");
+		mes("There are lots of men");
+		mes("but why should I do.");
+		mes("I'm not a slave.");
+		mes("Times have changed ~");
+		next;
+		if (select("What's the matter?", "Seem serious, ignore.") == 1) {
```
Comment (Asheraf) [inline npc/quests/arlim_quest.txt]: here check for the second option instead of the first.

File: npc/quests/arlim_quest.txt [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+			mes("actually, that's not wrong.");
+			mes("but please don't look at me");
+			mes("with that your eyes!");
+			next();
+			mes("[Errand Man]");
+			mes("ah!");
+			mes("That's not important thing.");
+			mes("whatever will you help me?");
+			next();
+			if (select("I will!", "I'm busy.") == 1) {
```
Comment (Asheraf) [inline npc/quests/arlim_quest.txt]: same as above

File: npc/quests/arlim_quest.txt [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+		}
+		emotion(e_otl);
+		mes("[Errand Man]");
+		mes("eeeheeeew~");
+		mes("Where am I here...");
+		mes("How can I");
+		mes("get all of them!");
+		close();
+	}
+	else if (.@pong == 1) {
```
Comment (Asheraf) [inline npc/quests/arlim_quest.txt]: `} else if (.@pong == 1) {`

File: npc/quests/arlim_quest.txt [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+				mes("Are you testing me?");
+				mes("I said 100 ^006400100 Jellopy^000000!");
+				next();
+				mes("[Errand Man]");
+				mes("^FF0000Poring^000000 or not anyway");
+				mes("If you are an adventurer");
+				mes("aren't you able to find it easily?");
+				close();
+			}
+			else {
```
Comment (Asheraf) [inline npc/quests/arlim_quest.txt]: `} else {`

File: npc/quests/arlim_quest.txt [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+				mes("[Errand Man]");
+				mes("um~");
+				mes("it seems correct approximately.");
+				mes("Thanks. Here is my gift.");
+				mes("it's popular one to adventurer");
+				mes("like you.");
+				close2();
+				completequest(9024);
+				delitem(Jellopy, 100);
+				getitem(Comp_Battle_Manual, 5);
```
Comment (Asheraf) [inline npc/quests/arlim_quest.txt]: maybe a custom weight check here?

File: npc/quests/arlim_quest.txt [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+		mes("what? what are you doing?!");
+		mes("100 ^006400100 Jellopy^000000!!");
+		mes("around this city");
+		mes("it is called ^FF0000Poring^000000");
+		mes("one of the brutal monster ");
+		mes("can get them!");
+		emotion(e_wah);
+		close();
+	}
+	else {
```
Comment (Asheraf) [inline npc/quests/arlim_quest.txt]: `} else {`

File: npc/quests/arlim_quest.txt (hunk 1 above)
Comment (Jedzkie) [inline npc/quests/arlim_quest.txt]: Fixed.

File: npc/quests/arlim_quest.txt (hunk 2 above)
Comment (Jedzkie) [inline npc/quests/arlim_quest.txt]: Fixed.

File: npc/quests/arlim_quest.txt (hunk 3 above)
Comment (Jedzkie) [inline npc/quests/arlim_quest.txt]: Fixed.

File: npc/quests/arlim_quest.txt (hunk 4 above)
Comment (Jedzkie) [inline npc/quests/arlim_quest.txt]: Fixed.

File: npc/quests/arlim_quest.txt (hunk 5 above)
Comment (Jedzkie) [inline npc/quests/arlim_quest.txt]: Fixed.

File: npc/quests/arlim_quest.txt (hunk 6 above)
Comment (Jedzkie) [inline npc/quests/arlim_quest.txt]: No need, since the weight of `Comp_Battle_Manual` is 0.

File: npc/quests/arlim_quest.txt (hunk 7 above)
Comment (Jedzkie) [inline npc/quests/arlim_quest.txt]: Fixed.

File: npc/quests/arlim_quest.txt (hunk 6 above)
Comment (Ridley8819) [inline npc/quests/arlim_quest.txt]: still the inventory could be full. But these checks are barely in aegis.

**Comment by skyleo** [comment]
Judging from the amount of grammar mistakes, this doesn't seem to be taken from iRO.
Who translated this?

**Comment by Jedzkie** [comment]
@skyleo its from my aegis, I can fix those wrong grammar later.

**Comment by Jedzkie** [comment]
Thanks @Smokexyz !

**Comment by Jedzkie** [comment]
@Asheraf is this good to merge now?

**Comment by Asheraf** [comment]
Yes unless there is more grammar mistakes to correct.

**Comment by skyleo** [comment]
There are still unresolved lines of my review.

**Comment by Jedzkie** [comment]
calling @Mysteries

**Comment by Mysteries** [comment]
I managed to fix majority of the broken English. It's the best I can do with the English translation that was given. Translation may not be exact.

**Comment by MishimaHaruna** [comment]
Is this ready? Can I request it to be rebased, squashing into 1 (or at most 2) commits before it's merged?

**Comment by dastgir** [comment]
If I remember well, there should be no next; between countitem and del/getitem.
Also delitem should go first (just in case someone learners new hack attempt), so that other remaining line won't execute.

**Pull request closed without merging**